### PR TITLE
use ObjectFieldValue to format postal fields (fixes issue of multiline addresses)

### PR DESCRIPTION
**Pull request by wipascal**

### CODE REVIEW

**Comment by wipascal** [comment]
I think the new version of lazy elements is genious!

**Comment by saemideluxe** [comment]
Right, it is very nice. But we might also encounter some issues along the way ;)
But I did not see you using in this PR, correct?

**Comment by wipascal** [comment]
> Right, it is very nice. But we might also encounter some issues along the way ;) But I did not see you using it in this PR, correct?

I used it ;) It made the refactoring a bit simpler :)

File: basxconnect/core/layouts/editperson/common/addresses.py [hunk 1]
```diff
@@ -111,36 +110,45 @@ def edit_heading(model: type):
...
         submitlabel=_("Save"),
     )
-    is_inactive = postal.valid_until and postal.valid_until < timezone.now().date()
+    is_inactive = hg.F(
+        lambda c: c["i"].valid_until and c["i"].valid_until < timezone.now().date()
+    )
     return R(
         C(
             hg.DIV(
                 postal.type,
```
Comment (wipascal) [inline basxconnect/core/layouts/editperson/common/addresses.py:131]: here

File: basxconnect/core/layouts/editperson/common/addresses.py [hunk 2]
```diff
@@ -111,36 +110,45 @@ def edit_heading(model: type):
...
+                    hg.BaseElement(" (", _("primary"), ")"),
+                ),
                 style="font-weight: bold; margin-bottom: 1rem;",
             ),
-            hg.DIV(postal.address, style="margin-bottom: 0.25rem;"),
+            hg.DIV(
+                ObjectFieldValue("address", object_contextname="i"),
+                style="margin-bottom: 0.25rem;",
+            ),
             hg.DIV(postal.postcode, " ", postal.city, style="margin-bottom: 0.25rem;"),
```
Comment (wipascal) [inline basxconnect/core/layouts/editperson/common/addresses.py:145]: here

Comment (wipascal) [reply basxconnect/core/layouts/editperson/common/addresses.py:145]: (etc)

Comment (saemideluxe) [reply basxconnect/core/layouts/editperson/common/addresses.py:145]: Ah, right. Nice. Btw. I also added support for item-accessor for things like ```hg.C("mydict")["key"]``` in one of the last commits. Maybe adding support for expressions or similar in the future.

**Review by saemideluxe — APPROVED** [review]

**Pull request merged**